src/main/java/core/basesyntax/services/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.services;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor(){
+        Color[] colors = Color.values();
+        Random rand = new Random();
```

Comment:
Better to use full names for variables
```suggestion
        Random random = new Random();
```

Reply:
Great suggestion! Get it.